_posts/platform/app/2000-01-01-log-drain.md

```diff
@@ -2,54 +2,139 @@
+[how to deploy the ELK
+stack](https://scalingo.com/articles/2018/02/23/running-the-elk-stack-on-scalingo.html)
+on the platform.
+
+Scalingo also support sending applications logs using the standard
+syslog protocol over UDP, TCP or TCP+TLS.
+
+{% note %}
+Adding a log drain to an application will not disable
+[scalingo logs](https://doc.scalingo.com/platform/app/logs)
+{% endnote %}
 
-## Third-party Provider integrations
+## Third-Party Provider Integrations
 
-Our system is also able to send logs to the following SaaS log processing providers:
+Our system is able to send logs to the following SaaS logs processing providers:
 
-* Scalingo hosted ELK stack ([how to setup]({% post_url platform/getting-started/2000-01-01-getting-started-with-elk %}))
-* Datadog
-* Logentries
-* Graylog
-* Papertrail
-* Syslog
+* [Scalingo hosted ELK stack](#scalingo-hosted-elk-stack)
+([how to setup]({% post_url platform/getting-started/2000-01-01-getting-started-with-elk %}))
+* [Datadog](#datadog)
+* [Logentries](#logentries)
+* [Graylog](#ovh-hosted-graylog)
+* [Papertrail](#papertrail)
+* [Syslog](#syslog-tcptls)
 
 ## Configuration of your Application on Scalingo
 
-As this feature is private beta, you cannot activate logs drain by yourself. You need to add an
-environment variable containing the token to the used logs management service, and contact the
-support so that we activate the logs drain. In a matter of minutes, your logs flow to the logs
-management service.
-
-The format of the environment variable depends on the provider:
-
-* Scalingo hosted ELK stack: `https://USER:TOKEN@logstash-app-name.osc-fr1.scalingo.io`
-* Datadog: `datadog://DD_API_KEY?region=REGION`. `DD_API_KEY` is the Datadog API key you can
-    find [here](https://app.datadoghq.com/account/settings#api). `REGION` value
-    is either `eu` or `us` depending on your Datadog account region.
-* Logentries: `https://webhook.logentries.com/noformat/logs/LOG-TOKEN`.
-    Instructions to get the `LOG-TOKEN` are available on [Logentries
-    documentation](https://docs.logentries.com/docs/http-post).
-* OVH hosted Graylog: `ovh://:TOKEN@REGION.logs.ovh.com:6514`. Instructions to
-    get the `TOKEN` and `REGION` variables are available on [OVH
-    documentation](https://docs.ovh.com/fr/logs-data-platform/quick-start/).
-* Papertrail: `syslog+tls://logsN.papertrailapp.com:XXXXX`. `logsN` and `XXXXX`
-    are the log host and port number shown under [log
-    destination](https://papertrailapp.com/account/destinations).
-* Syslog TCP+TLS: `tcp+tls://example.com:12345`
+The log drains configuration is done using the
+[CLI](https://doc.scalingo.com/cli).
+
+### Add a Log Drain
+
+To add a log drain management service you have to use the `log-drains-add`
+command. The parameters used depend on the chosen provider.
+
+Below, an exhaustive list of examples of additions for each of the supported
+vendors.
+
+It is possible to add multiple log drains to a single application
```

Comment:
These three paragraphs does not look nice:

![Screenshot from 2020-06-03 13-39-25](https://user-images.githubusercontent.com/1567651/83632528-b7c8e180-a59f-11ea-9fda-22bd6fc19246.png)

Can you try to reorganize so that it's less ugly?

For example, The sentence " It is possible to add multiple log drains to a single application " should be included in the paragraph starting with "To add a log drain management service you" IMHO.